docker/Dockerfile

```diff
@@ -1,24 +1,27 @@
 ARG VERSION
 
-FROM ghcr.io/deephaven/server:${VERSION}
+FROM ghcr.io/deephaven/server-sklearn:${VERSION}
```

Comment:
I would do this differently.  I would make the base image an argument so that we don't always deploy sklearn when users don't want it.

The suggested ARGs will require changing `docker-compose.yml` and `deephaven_ib_docker.sh`.